job/task.sh

```diff
@@ -1,12 +1,19 @@
 #
 # The script to run the user provided commands is generated here
 #
+export SUBSCRIPTION_PRIVATE_KEY="$BUILD_DIR/secrets/00_sub"
```

Comment:
this is making `SUBSCRIPTION_PRIVATE_KEY` to be available in the scope of `runSh` commands.

example: if user has following runSh yml, this value will be printed.

```yml
  - name: old-min-sh
    type: runSh
    steps:
      - TASK:
        - script: echo "$SUBSCRIPTION_PRIVATE_KEY"
```

 Is it expected behaviour ?